cmd/secrets-store-csi-driver/main.go

```diff
@@ -24,15 +24,16 @@ import (
 	_ "net/http/pprof" // #nosec
 	"time"
 
-	"sigs.k8s.io/secrets-store-csi-driver/pkg/cache"
+	"sigs.k8s.io/controller-runtime/pkg/cache"
```

Comment:
Let's move the import to after line 39 and run `gofmt`. The imports should be grouped

```
import (
  stdlib

  internal packages

  external packages
)
```